packages/clay-tabs/docs/index.js

```diff
@@ -63,6 +63,66 @@ const Tabs = () => {
 	return <Editor code={tabsCode} imports={tabsImportsCode} scope={scope} />;
 };
 
+const tabsLightImportsCode = `import ClayTabs from '@clayui/tabs';
+`;
+
+const tabsLightCode = `const Component = () => {
+	const [active, setActive] = useState(0);
+
+	return (
+		<>
+			<ClayTabs active={active} displayType="light" onActiveChange={setActive}>
+				<ClayTabs.Item
+					innerProps={{
+						'aria-controls': 'tabpanel-1',
+					}}
+				>
+					Tab 1
+				</ClayTabs.Item>
+				<ClayTabs.Item
+					innerProps={{
+						'aria-controls': 'tabpanel-2',
+					}}
+				>
+					Tab 2
+				</ClayTabs.Item>
+				<ClayTabs.Item
+					innerProps={{
+						'aria-controls': 'tabpanel-3',
+					}}
+				>
+					Tab 3
+				</ClayTabs.Item>
+			</ClayTabs>
+			<ClayTabs.Content activeIndex={active} fade>
+				<ClayTabs.TabPane aria-labelledby="tab-1">
+					1. Proin efficitur imperdiet dolor, a iaculis orci lacinia eu.
+				</ClayTabs.TabPane>
+				<ClayTabs.TabPane aria-labelledby="tab-2">
+					2. Proin efficitur imperdiet dolor, a iaculis orci lacinia eu.
+				</ClayTabs.TabPane>
+				<ClayTabs.TabPane aria-labelledby="tab-3">
+					3. Proin efficitur imperdiet dolor, a iaculis orci lacinia eu.
+				</ClayTabs.TabPane>
+			</ClayTabs.Content>
+		</>
+	);
```

Comment:
@pat270 We can update these examples to use the new compositing pattern now

Reply:
@matuzalemsteles I'm not sure what you mean by new compositing pattern. Can you point me to an example?

Reply:
You can see the example in the storybook and do the same here https://storybook.clayui.com/?path=/story/design-system-components-tabs--new-default